tabula/wrapper.py

```diff
@@ -8,74 +8,84 @@
+      guess (bool, optional): Guess the portion of the page to analyze per page.
+      area (:obj:`list` of :obj:`float`, optional): Portion of the page to analyze(top,left,bottom,right). Example: [269.875,12.75,790.5,561]. Default is entire page
+      spreadsheet (bool, optional): Force PDF to be extracted using spreadsheet-style extraction (if there are ruling lines separating each cell, as in a PDF of an Excel spreadsheet)
+      nospreadsheet (bool, optional): Force PDF not to be extracted using spreadsheet-style extraction (if there are ruling lines separating each cell, as in a PDF of an Excel spreadsheet)
+      password (bool, optional): Password to decrypt document. Default is empty
+      silent (bool, optional): Suppress all stderr output.
 
-  Returns:
-    Extracted pandas DataFrame.
-  '''
+    Returns:
+      Extracted pandas DataFrame.
+    '''
 
-  __options = []
-  # handle options described in string for backward compatibility
-  __options += shlex.split(options)
+    __options = []
+    # handle options described in string for backward compatibility
+    __options += shlex.split(options)
 
-  # parse options
-  if pages:
-    __pages = pages
-    if type(pages) == int:
-      __pages = str(pages)
-    elif type(pages) in [list, tuple]:
-      __pages = ",".join(map(str, pages))
+    # parse options
+    if pages:
+        __pages = pages
+        if type(pages) == int:
+            __pages = str(pages)
+        elif type(pages) in [list, tuple]:
+            __pages = ",".join(map(str, pages))
 
-    __options += ["--pages", __pages]
+        __options += ["--pages", __pages]
 
-  if guess:
-    __options.append("--guess")
+    if guess:
+        __options.append("--guess")
 
-  if area:
-    __area = area
-    if type(area) in [list, tuple]:
-      __area = ",".join(map(str, area))
+    if area:
+        __area = area
+        if type(area) in [list, tuple]:
+            __area = ",".join(map(str, area))
 
-    __options += ["--area", __area]
+        __options += ["--area", __area]
 
-  if spreadsheet:
-    __options.append("--spreadsheet")
+    if spreadsheet:
+        __options.append("--spreadsheet")
 
-  if nospreadsheet:
-    __options.append("--no-spreadsheet")
+    if nospreadsheet:
+        __options.append("--no-spreadsheet")
 
-  if password:
-    __options += ["--password", password]
+    if password:
+        __options += ["--password", password]
 
-  if silent:
-    __options.append("--silent")
+    if silent:
+        __options.append("--silent")
 
-  args = ["java", "-jar", jar_path] + __options + [input_path]
+    if outfile:
+        __options += ["--outfile", outfile]
 
-  output = subprocess.check_output(
-    args,
-    stderr=subprocess.STDOUT)
+    args = ["java", "-jar", jar_path] + __options + [input_path]
+    stdoutput = subprocess.check_output(
```

Comment:
I think output and outfile are ambiguous when placed together. So i changed output to stdoutput to clarify